pulp_smash/config.py

```diff
@@ -73,6 +81,7 @@ def _get_pulp_3_api_role():
 P2_OPTIONAL_ROLES = {
     'pulp cli',
     'squid',
+    'content',
```

Comment:
Is this one supposed to be part of Pulp 2? Can we update this to make a bit clear about what Pulp version?

Reply:
I will get this code locally and review. Just a minor comment when skimming through the code.

Reply:
@kersommoura naming problem, both pulp2 and pulp3 uses the same `P2_` prefixed variables.

Reply:
@rochacbruno, as we talked this change will require changes in the CLI as well.